ice-phishing/src/agent.js

```diff
@@ -667,7 +842,9 @@ const provideHandleTransaction = (provider) => async (txEvent) => {
               if (!balance.eq(0)) return;
             }
           }
-          findings.push(createHighNumTransfersAlert(txFrom, transfers[txFrom]));
+          counters.detectedTransfers += 1;
```

Comment:
```suggestion
          counters.detectedTransfers += transfers[txFrom].length;
```
Suggestion to use alert's underlying amount of transfers, rather than `1`. (I think this is the correct `.length`)